tests/parsing/test_full_telemetry_parsing.py

```diff
@@ -114,20 +114,20 @@ def invalid_packet_header() -> PacketHeader:
 
 
 # Test valid header
-def test_is_valid_hdr(valid_packet_header: PacketHeader, approved_callsigns: dict[str, str]) -> None:
-    assert is_valid_packet_header(valid_packet_header, approved_callsigns)
+def test_is_approved_pkt_hdr(valid_packet_header: PacketHeader, approved_callsigns: dict[str, str]) -> None:
+    assert is_approved_packet_header(valid_packet_header, approved_callsigns) is None
 
 
 # Test a invalid header: unapproved call sign
 # def test_is_invalid_hdr1(non_approved_callsign: PacketHeader, approved_callsigns: dict[str, str]) -> None:
-#     assert not (is_valid_packet_header(non_approved_callsign, approved_callsigns))
+#     assert not (is_approved_packet_header(non_approved_callsign, approved_callsigns))
 
 
 # # Test invalid header: version number 0
 # def test_is_invalid_hdr2(version_num_zero: PacketHeader, approved_callsigns: dict[str, str]) -> None:
-#     assert not (is_valid_packet_header(version_num_zero, approved_callsigns))
+#     assert not (is_approved_packet_header(version_num_zero, approved_callsigns))
 
 
 # # Test invalid header: non approved callsign and incorrect version number
 # def test_is_invalid_hdr3(invalid_packet_header: PacketHeader, approved_callsigns: dict[str, str]) -> None:
-#     assert not (is_valid_packet_header(invalid_packet_header, approved_callsigns))
+#     assert not (is_approved_packet_header(invalid_packet_header, approved_callsigns))
```

Comment:
Instead of checking that this function returns none which doesn't really do much, can you have the tests check that the logger output the correct message upon receiving an approved or unapproved callsign?

https://pytest-with-eric.com/fixtures/built-in/pytest-caplog/#Pytest-Caplog-Example